frontend/src/Theme.css

```diff
@@ -18,6 +18,7 @@
 	--main-text-accent-colour: #73d1d6;
 	--error-colour: #c41565;
 	--error-background-colour: #faebf2;
+	--error-colour-light: #ff75af;
```

Comment:
I rather like this colour! I am assuming that our "standard" error colour is not contrasting enough on the dark background.

Reply:
yep! I used https://www.learnui.design/tools/accessible-color-generator.html to find the closest match with good contrast